charts/fybrik/files/taxonomy/datacatalog.json

```diff
@@ -43,6 +43,34 @@
         }
       }
     },
+    "DeleteAssetRequest": {
+      "type": "object",
+      "required": [
+        "destinationCatalogID"
+      ],
+      "properties": {
+        "destinationAssetID": {
+          "description": "Asset ID of the to-be deleted asset",
+          "type": "string"
+        },
+        "destinationCatalogID": {
+          "description": "The destination catalog id in which the asset to be deleted exists",
+          "type": "string"
+        }
+      }
+    },
+    "DeleteAssetResponse": {
+      "type": "object",
+      "required": [
+        "status"
```

Comment:
Why would we need anything except for the returned code?

Reply:
Yes, I'd change it to an optional message

Reply:
Thanks. Could you give some pointers to the error codes we use for delete operation? I can provide a required error code field in the `DeleteAssetResponse` object while making `Status` field optional. @shlomitk1

Reply:
@rohithdv , see https://www.restapitutorial.com/lessons/httpmethods.html

Reply:
@roytman I had one query. These status codes will be anyways being sent from the `deleteAsset` implementation. So we need not repeat these codes in the response object right ? I am not sure what should be provided in the `DeleteAssetResponse` object. Could you elaborate on that? @shlomitk1

Reply:
An optional error message